worktree_status.go

```diff
@@ -370,7 +370,12 @@ func (w *Worktree) doAdd(path string, ignorePattern []gitignore.Pattern, skipSta
 		}
 	}
 
+	// Relative paths only; see: https://github.com/go-git/go-git/issues/1460
 	path = filepath.Clean(path)
+	if filepath.IsAbs(path) {
+		root := w.Filesystem.Root()
+		path, _ = filepath.Rel(root, filepath.Join(root, ".", path))
+	}
```

Comment:
Hmm.  This works on OSX and Linux, and fails on Windows.  I don't have access to any Windows boxes to figure out what's going wrong, can anybody else add any colour here?